### PR TITLE
Open source /allocation/compute

### PR DESCRIPTION
**Pull request by nikovacevic**
Requires https://github.com/kubecost/kubecost-cost-model/pull/360

## Changes
- Move `/model/allocation/compute` to open source

## Testing
- Manual on gcp-niko

### CODE REVIEW

**Review by mbolt35 — APPROVED** [review]
Looks good to me! Hype for more open source! 🥇

File: pkg/costmodel/aggregation.go [hunk 1]
```diff
@@ -2114,6 +2114,136 @@ func (a *Accesses) AggregateCostModelHandler(w http.ResponseWriter, r *http.Requ
...
+	// Window is a required field describing the window of time over which to
+	// compute allocation data.
+	window, err := kubecost.ParseWindowWithOffset(qp.Get("window", ""), env.GetParsedUTCOffset())
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Invalid 'window' parameter: %s", err), http.StatusBadRequest)
+	}
+
+	// Block is an optional parameter that defines the duration per-block, i.e.
+	// the window for an AllocationSet, of the AllocationSetRange to be
+	// computed. Defaults to the window size, making one block.
```
Comment (SheepGotoHeaven) [inline pkg/costmodel/aggregation.go:2187]: Does "Block" need to be replaced with "Step" in these comments?

**Review by SheepGotoHeaven — APPROVED** [review]
LGTM!

File: pkg/costmodel/aggregation.go (hunk 1 above)
Comment (nikovacevic) [reply pkg/costmodel/aggregation.go:2187]: Indeed, good catch!

**Pull request merged**